dspace-server-webapp/src/main/java/org/dspace/app/rest/converter/MockMetadataConverter.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.converter;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+import org.dspace.app.rest.model.MetadataRest;
+import org.dspace.app.rest.model.MetadataValueRest;
+import org.dspace.content.MetadataValue;
+import org.dspace.mock.MetadataValueDTO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+/**
+ * Converter to translate between lists of domain {@link MetadataValue}s and {@link MetadataRest} representations.
+ */
+@Component
+public class MockMetadataConverter implements Converter<List<MetadataValueDTO>, MetadataRest> {
```

Comment:
This Converter should be renamed so that it no longer includes the term `Mock`.  Perhaps this should be renamed `MetadataValueDTOListConverter`, since this converts a List of MetadataValueDTO objects.